src/OrchardCore.Modules/OrchardCore.ContentTypes/ViewModels/ContentDefinitionStepViewModel.cs

```diff
@@ -0,0 +1,9 @@
+namespace OrchardCore.ContentTypes.ViewModels
+{
+    public class ContentDefinitionStepViewModel
```

Comment:
There should be a "All" option that would always export everything. It's different than having everything selected, because there could be new types overtime, and the "All" would still pick them up. Useful for backups.